src/components/EmojiAvatar/EmojiAvatar.styled.ts

```diff
@@ -0,0 +1,61 @@
+/* Internal dependencies */
+import {
+  styled,
+  css,
+  absoluteCenter,
+  centeredBackgroundImage,
+} from 'Foundation'
+import { InterpolationProps } from 'Types/Foundation'
+import { EmojiAvatarSize } from './EmojiAvatar.types'
+
+const AVATAR_STATUS_GAP = 2
+const WRAPPER_SIZE = 24
+
+interface IconProps extends InterpolationProps {
+  unClickable?: boolean
+  size?: EmojiAvatarSize
+  privateChat?: boolean
+}
+
+export const Wrapper = styled.div`
```

Comment:
```suggestion
export const Wrapper = styled.button`
```

이 컴포넌트가 클릭 가능한 컴포넌트라면 `Wrapper` 컴포넌트는 `button` 엘리먼트가 되는게 맞을 거 같아요. 클릭 가능 여부에 따라 `as` 속성을 통해 `div` <-> `button` 전환이 되었으면 좋겠습니다.